platform/android/java/app/build.gradle

```diff
@@ -205,36 +205,54 @@ android {
 }
 
 task copyAndRenameDebugApk(type: Copy) {
+    // No need for up-to-date checks for the copy-and-rename operation
```

Comment:
```suggestion
```
No need to repeat the message IMO

Reply:
Even though the strings are the same, they are serving different purposes which is why I have them both as comment and as the string parameter for the `doNotTrackState(...)` method:
- The comment is an indication to engine developers about why we need to use the `doNotTrackState(...)`. It doesn't have to be there, it's more of a stylistic preference as I prefer to be overly verbose and descriptive of the code I write.
- The string parameter is for end users reading the output of the gradle build. That parameter can be different from the comment, I couldn't think of anything better so I re-used the comment. Suggestions welcomed!

Reply:
I'd suggest describing what's going on if the comment should remain when the same as the message itself, it doesn't describe really what's going on I'd say

Reply:
Sounds good! I've updated the comment to describe what's going on.
Let me know if that addresses your feedback!

Reply:
LGTM!